packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -24,6 +24,7 @@
   - [Publishing HTTP APIs](#publishing-http-apis)
   - [Custom Domain](#custom-domain)
   - [Metrics](#metrics)
+  - [VPC Link](#vpc-link)
```

Comment:
Add a section after vpc link called 'private integration', mention an introduction sentence on it and then point folks at the new module.